From 373be24cfd721106e5aa7d5a60ee765e911cc0b3 Mon Sep 17 00:00:00 2001
From: 2kybe3 <kybe@kybe.xyz>
Date: Sun, 15 Feb 2026 15:24:38 +0100
Subject: [PATCH] remove max connection per server limit

---
 src/OptionHandlerFactory.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/OptionHandlerFactory.cc b/src/OptionHandlerFactory.cc
index 000426b6..2eac3ad3 100644
--- a/src/OptionHandlerFactory.cc
+++ b/src/OptionHandlerFactory.cc
@@ -438,7 +438,7 @@ std::vector<OptionHandler*> OptionHandlerFactory::createOptionHandlers()
   {
     OptionHandler* op(new NumberOptionHandler(PREF_MAX_CONNECTION_PER_SERVER,
                                               TEXT_MAX_CONNECTION_PER_SERVER,
-                                              "1", 1, 16, 'x'));
+                                              "1", 1, -1, 'x'));
     op->addTag(TAG_BASIC);
     op->addTag(TAG_FTP);
     op->addTag(TAG_HTTP);
-- 
2.52.0

